scripts/deploy.ts

```diff
@@ -17,6 +19,17 @@ const argv = require("minimist")(process.argv.slice(2), {
     --pubkeys 06642b1af0ec2f7369126b3e45aaf11a050a26e2111150b319c4ca2f4a8d48c62442e01b651ce469632972f06c4d5d092da2cde3d42c0ead5ebeed8646d72b3f1f09c41c70cbdd8779920b4a3b15e6c7c518c5fdb93fdaed7a49c59b1f4d1e210aafa7a17239c9df9053ca4e71f10fb9f7b0f219e12e300676a756463253c287,2a7575b1bddf2c2b25f976788baae059ea410f589f0c244e1554d6f6f5398b6a1b997670d19c26c33ad1c61e21bd2742565518e0deafb105e7718de983bf757a0770cb4889aabb6f4897534f4770fce4b76f39b2cc8ed63b22e55ebf7e87a96001bbabd11ca55585bc0c016ad058a69c682cbe102a4b91f6084c71c857509dac
 */
 
+function getDefaultGenesisRoot(parameters: DeploymentParameters) {
+    const stateTree = StateTree.new(parameters.MAX_DEPTH);
+    const LARGE_AMOUNT = 1000000000;
+    const state0 = State.new(0, 0, LARGE_AMOUNT, 0);
```

Comment:
why do we initialise with LARGE_AMOUNT instead of `0`?

Reply:
So it might be the case that the coordinator wants to spend some token ASAP. Note that we now don't require special states for Mass migration or create2transfer. The default genesis states are created just for convenience.

Reply:
This will make the `leaf_hash != defaultHashes(0)` which will lead to inconsistencies on the client. If this change is not required by some feature then think we should skip it.

Reply:
Sounds good to me.